front/components/agent_builder/triggers/ScheduleEditionModal.tsx

```diff
@@ -42,6 +42,16 @@ function formatTimezone(timezone: string): string {
   return `${city} (${timezone})`;
 }
 
+function extractErrorMessage(error: unknown): string {
+  if (typeof error === "object" && error !== null && "error" in error) {
+    const innerError = error.error;
+    if (typeof innerError === "object" && innerError !== null && "message" in innerError) {
+      return innerError.message as string;
+    }
+  }
```

Comment:
Ugly but I could not make the normalizeError utility work as I don't have an Error object
Let me know if there is a better way